pkg/config/config.go

```diff
@@ -69,6 +70,9 @@ func (c Config) Blockchain() Blockchain {
 // config will be updated based on the provided relative path.
 func Load(path string, netMode netmode.Magic, relativePath ...string) (Config, error) {
 	configPath := fmt.Sprintf("%s/protocol.%s.yml", path, netMode)
+	if _, err := os.Stat(configPath); os.IsNotExist(err) {
+		return getEmbeddedConfig(netMode)
+	}
```

Comment:
I think it should be placed to `LoadFile`. We don't need code duplications between `Load` and `LoadFile`. See https://github.com/nspcc-dev/neo-go/pull/3477#discussion_r1655117101.